Framework/LiveData/src/FileEventDataListener.cpp

```diff
@@ -148,6 +151,17 @@ std::shared_ptr<Workspace> FileEventDataListener::extractData() {
 
   m_runNumber = chunk->getRunNumber();
 
+  if (m_loaderName == "LoadEventNexus") {
+    // Scale the proton charge by the number of chunks
+    TimeSeriesProperty<double> *pcharge = chunk->mutableRun().getTimeSeriesProperty<double>("proton_charge");
+    auto values = pcharge->valuesAsVector();
+    for (auto &value : values) {
```

Comment:
You rewrote [`std::transform`](https://en.cppreference.com/w/cpp/algorithm/transform)